features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/NewOverview/Configuration.jsx

```diff
@@ -138,32 +138,38 @@ function Configuration(props) {
                                 {api.context && <React.Fragment>{api.context}</React.Fragment>}
                             </Typography>
                             {/* Version */}
-                            <Typography component='p' variant='subtitle2' className={parentClasses.subtitle}>
-                                <FormattedMessage
-                                    id='Apis.Details.NewOverview.Configuration.version'
-                                    defaultMessage='Version'
-                                />
-                            </Typography>
-                            <Typography component='p' variant='body1'>
-                                {api.version && <React.Fragment>{api.version}</React.Fragment>}
-                            </Typography>
+                            {(api.apiType === API.CONSTS.API) &&
+                                <React.Fragment>
```

Comment:
When conditionally rendering, shouldn't the content be wrapped with `()` if it's multiline? 

Similarly in, L156 as well.

Reply:
fixed with https://github.com/wso2/carbon-apimgt/commit/a5b4d09fd25478c04985b9e5f2eea62f56fed44d